src/get-data.ts

```diff
@@ -40,15 +49,34 @@ const getData = async (): Promise<PRData> => {
   if (info.repository && info.sender) {
     prData.actor = info.sender.login;
   } else {
-    throw new Error('Issue with context');
+    throw new Error('Error: Issue with context');
```

Comment:
We have more information here. If we get here we know that one or both of `repository` and `sender` are undefined so we should say that in the error.